src/WebRtcTransport.ts

```diff
@@ -16,7 +16,7 @@ export type WebRtcTransportOptions =
 	 * Listening IP address or addresses in order of preference (first one is the
 	 * preferred one).
 	 */
-	listenIps: TransportListenIp[] | string[];
+	listenIps: Array<TransportListenIp | string>;
```

Comment:
Now I understand this :)
However, can we do this instead?:

```suggestion
	listenIps: (TransportListenIp | string)[];
```

I've tested it and seems to be the very same.

Reply:
You probably can, but that is not idiomatic in TypeScript to do

Reply:
Ok, but I prefer to not introduce exceptions. The whole TS code in mediasoup right now uses `[]` syntax instead of `Array`. We can change the whole thing in the future.

Reply:
tslint's default codestyle argues that `T[]` is for simple types and `Array<T | R>` or `Array<T<R>>` is for complex cases

Reply:
`tslint` is deprecated. Using `eslint` with `@typescript-eslint/xxxx` packages does not complain about it (tested right now).

Reply:
Well, if you're so opposed to it... `(TransportListenIp | string)[]` looks completely alien to me.

Reply:
I'm not against it. I just mean that I don't want to use two separate syntaxes. We can move all `[]` to `Array<>` or keep using `[]`. I just don't want mixed code. Since this is just cosmetic it can be done later, right?

Reply:
I don't see why `string[]` needs to change to `Array<string>`, it is also too verbose. Anyway, yes, that is just cosmetic, the point is to have an array as outer data structure.

Reply:
Ok, I promise I'll revisit this when it comes to modernize TS stuff.

Reply:
so can you please also change this to `listenIps: Array<TransportListenIp | string>;` for now?

Reply:
sorry, I mean:

`listenIps: (TransportListenIp | string)[];`

Reply:
Done